### DISCUSSION TITLE
notifications:inbox is showing my that I have 1 unread notifications, but I cannot see one!

### DISCUSSION THREAD

**Discussion by tullo** [community/community#48566] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

notifications:inbox is showing my that I have 1 unread notification, but I cannot see one!

The 1 shows up for repo:caddyserver/caddy
- Clicking on that repo reveals no unread notification
- Same for clicking the "All" tab
- It has been like that for several months now and I thought it would eventually go away

Is that caused by wrongdoing on my side in some way?
Or could this be a bug?

**Comment by radar-blip** [comment, marked as answer]
Sounds like a ghost notification bug.
You can try steps in this thread to fix:

https://github.com/orgs/community/discussions/6874#discussioncomment-3593547
